.werft/jobs/build/job-config.ts

```diff
@@ -136,11 +139,15 @@ export function jobConfig(werft: Werft, context: any): JobConfig {
     }
 
     werft.log("job config", JSON.stringify(jobConfig));
-    werft.rootSpan.setAttributes(Object.fromEntries(Object.entries(jobConfig).map((kv) => {
+    const globalAttributes = Object.fromEntries(Object.entries(jobConfig).map((kv) => {
         const [key, value] = kv
         return [`werft.job.config.${key}`, value]
-    })))
-    werft.rootSpan.setAttribute('werft.job.config.branch', context.Repository.ref)
+    }))
+    globalAttributes['werft.job.config.branch'] = context.Repository.ref
+    werft.addGlobalAttributes(globalAttributes)
```

Comment:
Matching the [suggestion below](https://github.com/gitpod-io/gitpod/pull/8797/files#r829792620)

```suggestion
    werft.addAttributes(globalAttributes)
```